var/spack/repos/builtin/packages/r-rcurl/package.py

```diff
@@ -22,10 +22,11 @@ class RRcurl(RPackage):
     url      = "https://cloud.r-project.org/src/contrib/RCurl_1.95-4.8.tar.gz"
     list_url = "https://cloud.r-project.org/src/contrib/Archive/RCurl"
 
+    version('1.98-1.2', sha256='5d74a0cdc3c5684b0348b959f67039e3c2a5da2bbb6176f6800a94124895a7a8')
     version('1.95-4.12', sha256='393779efafdf40823dac942a1e028905d65c34f3d41cfd21bcd225e411385ff4')
     version('1.95-4.8', sha256='e72243251bbbec341bc5864305bb8cc23d311d19c5d0d9310afec7eb35aa2bfb')
 
-    depends_on('r@3.0.0:', type=('build', 'run'))
+    depends_on('r@3.4.0:', type=('build', 'run'))
```

Comment:
I would do something like:
```python
depends_on('r@3.4.0:', when='@1.98:', type=('build', 'run'))
depends_on('r@3.0.0:', type=('build', 'run'))
```
That way this package can still be built with older versions of R. If you can figure out the exact version of RCurl that changed the R version dependency, that would be even better.

Reply:
Thanks for the tip, fix sent.